tests/features/steps/wrappers.py

```diff
@@ -4,17 +4,44 @@
 import re
 import pexpect
 from pgcli.main import COLOR_CODE_REGEX
+import textwrap
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
 
 
 def expect_exact(context, expected, timeout):
+    timedout = False
     try:
         context.cli.expect_exact(expected, timeout=timeout)
-    except:
+    except pexpect.exceptions.TIMEOUT:
```

Comment:
This seems to cause #996 to me - pexpect 3.0+ does not use `pexpect.exceptions`.